protocol/src/lib.rs

```diff
@@ -370,6 +370,11 @@ pub enum Message {
         flush_number: u64,
         gen_number: u64,
         snapshot_details: Option<SnapshotDetails>,
+        /*
+         * The ending extent where a flush should stop.
+         * This value is unique per downstairs.
```

Comment:
This is only true during repair, right?

Reply:
the comment [here](https://github.com/oxidecomputer/crucible/pull/651/files#diff-6d9d1c0cbb1fdf3f309bac3c91b38560a952bf9e17b4e64b6d1e04a13f160b1bR2490) answers my question